src/dedicatedCloud/servicePacks/dedicatedCloud-servicePacks.v6.service.js

```diff
@@ -0,0 +1,36 @@
+angular
+    .module("ovh-api-services")
+    .service("OvhApiDedicatedCloudServicePacksV6", function ($resource, $cacheFactory) {
+        "use strict";
+
+        var queryCache = $cacheFactory("OvhApiDedicatedCloudServicePacksV6Query");
+        var cache = $cacheFactory("OvhApiDedicatedCloudServicePacksV6");
+
+        var servicePacksResource = $resource("/dedicatedCloud/:serviceName/servicePacks", {
```

Comment:
Is there any reason to not specify this following URL instead: `/dedicatedCloud/:serviceName/servicePacks/:name`?

Reply:
No, and it does seem like having the most specific URL is the most common practice in `ovh-api-service` so I've updated !